www/officers/coi.cgi

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env ruby
+PAGETITLE = "Conflict of Interest Affirmations"
```

Comment:
Add a suitable # Wvisible:... comment?

Reply:
www/officers/acreq.cgi:PAGETITLE = "Apache Account Submission Helper Form" # Wvisible:infra accounts
www/officers/board-stats.cgi:PAGETITLE = "Board Meeting Statistics since 2007" # Wvisible:meeting
www/officers/index.cgi:PAGETITLE = "Overview of Whimsy Tools for Officers" # Wvisible:meeting
www/officers/list-traffic.cgi:PAGETITLE = "Board@ Mailing List Statistics" # Wvisible:board
www/officers/surveys.cgi:PAGETITLE = "Whimsy Member and Officer Surveys" # Wvisible:members
www/officers/test.cgi:PAGETITLE = "Whimsy Environment Tester" # Wvisible:tools

I'm afraid I cannot tell what the right #W visible is for coi.cgi ...